src/test/java/wooteco/subway/domain/PathTest.java

```diff
@@ -0,0 +1,39 @@
+package wooteco.subway.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import wooteco.subway.domain.section.Section;
+import wooteco.subway.domain.station.Station;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PathTest {
+
+    private Path path;
+
+    @BeforeEach
+    void setPath() {
+        List<Station> stationList = new ArrayList<>(List.of(new Station(1L, "강남역"), new Station(2L, "선릉역"), new Station(3L, "잠실역")));
+
+        List<Section> sectionList = new ArrayList<>(List.of(new Section(1L, 1L, 1L, 2L, 10), new Section(2L, 1L, 2L, 3L, 10)));
```

Comment:
```suggestion
        List<Station> stationList = new ArrayList<>(List.of(강남, 선릉, 잠실));

        List<Section> sectionList = new ArrayList<>(List.of(강남_선릉_10, 선릉_잠실_10));
```

테스트 코드도 조금 더 신경쓴다면 가독성을 크게 향상시킬 수 있습니다. 이런 Path, Fare과 같은 도메인들은 테스트 코드가 정말 중요한데요, 가독성을 올려서 유지보수가 편해지는 코드를 작성하는데 집중해보시면 좋을 것 같습니다 😄

Reply:
따로 도메인을 빼두니, 의미가 더 명확해 진 것 같습니다.
`가독성 좋은 테스트가 중요하다` 잘 기억해두겠습니다.
감사합니다 😄